### PR TITLE
Popup verification before sending email

### LINKED ISSUE #48: Add popup verification after pressing send button
- [ ] Add a popup when clicking send that asks something like 'Are you sure you want to send all emails?' and then a yes and no button 
- [ ] If yes is selected, the emails get sent and the modal is closed
- [ ] If no is selected, the emails do not get sent and the modal is closed
- [ ] Use the material ui component for popups https://mui.com/material-ui/react-dialog/

### PR DESCRIPTION
**Pull request by karynayen**
Closes #48 

Add a popup when clicking send that asks the user:  'Are you sure you want to send all emails?' and then a yes and no button. Emails are only sent if the 'yes' button is pressed.

https://user-images.githubusercontent.com/37753574/178343063-2bc74e9d-12fc-478e-a1fb-e33b9e30db3a.mov

### CODE REVIEW

File: pages/emailSender.tsx [hunk 1]
```diff
@@ -340,12 +351,28 @@ const EmailSender: NextPage = () => {
...
               Send!
             </StyledButton>
+            <Dialog
+              open={open}
+              onClose={handleClose}
+              aria-labelledby="alert-dialog-title"
+              aria-describedby="alert-dialog-description"
+            >
+              <DialogTitle id="alert-dialog-title">
+                {"Are you sure you want to send all emails?"}
```
Comment (jzhzhang66) [inline pages/emailSender.tsx:367]: We don't need to add the curly brackets and quotes here - you can just change to Are you sure you want to send all emails?

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Looks amazing!!!! You're killin it karyna:)))) 

Just one small comment and I was also wondering if your pre-commit hook is working? 

There's some eslint errors - when you run yarn run lint or commit your stuff do you see any errors? 
Screen Shot 2022-07-11 at 4 22 10 PM

**Comment by karynayen** [comment]
omg I finally fixed the pre-commit issue, it turns out I had an extra node-modules directory folder outside of the main directory that was causing an issue! I just had to delete it and it worked!

**Comment by karynayen** [comment]
hmm also I ended up re-cloning and reinstalling everything and the version types changed... is that ok?

**Review by jzhzhang66 — APPROVED** [review]
NICE JOB KARYNAAAA<33333

File: pages/emailSender.tsx [hunk 2]
```diff
@@ -340,12 +351,30 @@ const EmailSender: NextPage = () => {
...
               width="medium"
               disabled={finalMessages.length === 0}
             >
               Send!
             </StyledButton>
+            <Dialog
+              open={open}
+              onClose={handleClose}
+              aria-labelledby="alert-dialog-title"
+              aria-describedby="alert-dialog-description"
```
Comment (deanframe) [inline pages/emailSender.tsx:364]: We don't have a DialogContent or DialogContentText, so no need for this `aria-describedby` label

**Review by deanframe — APPROVED** [review]
Looks mostly great! Fix this then merge :D

**Pull request merged**